ember/app/components/project-detailed/component.js

```diff
@@ -28,4 +29,14 @@ export default class ProjectDetailedComponent extends Component {
     }
     this.store.findRecord('project', this.args.project.id);
   });
+
+  deleteProject = dropTask(async () => {
+    try {
+      this.args.project.destroyRecord();
+      this.router.transitionTo('projects.index');
+      this.notification.success('Project deleted successfully');
```

Comment:
The notification should occur before the transition as the transition migh cancel this task before it can show the notification

Reply:
done here: https://github.com/adfinis/Outdated/pull/71/commits/85720bce36a97b047bb2ff050cba7842d284c780